### ISSUE TITLE
default branch should be called `main`

### ISSUE THREAD

**Issue by leobenkel** [leobenkel/PSF-LoginServer#7]
current default branch is `master` but it should be called `main`

**Comment by Fate-JH** [comment]
This project was started before GitHub renamed the default project branches - when `master` was the default.

**Comment by leobenkel** [comment]
yes, but its pretty straightforward to update it. so why not? :)

**Comment by Fate-JH** [comment]
1. Changing it offers no substantial improvements to the project, either at compile-time or run-time.
2. The core justification for changing default branch names is based on non-technical considerations that can be argued do not even properly frame or accomplish its own goals due to the scope, and is difficult to justify even in a style guide,

As someone who regularly and regrettably makes unfortunately necessary gargantuan changes that were originally intended to be small feature extensions, my main policy is to not change something that does not need to be changed.  It doesn't always work out but I try to stick with it as much as possible.  `master` will become `main` one day, but when it is the easier solution to a different issue, when it is absolutely necessary.
